src/Controller/CRUDController.php

```diff
@@ -93,7 +93,7 @@ public static function getSubscribedServices(): array
             'sonata.admin.audit.manager' => AuditManagerInterface::class,
             'sonata.admin.object.manipulator.acl.admin' => AdminObjectAclManipulator::class,
             'sonata.admin.request.fetcher' => AdminFetcherInterface::class,
-            'sonata.exporter.exporter' => '?'.Exporter::class,
+            'sonata.exporter.exporter' => '?'.ExporterInterface::class,
```

Comment:
I think some change here:
https://github.com/sonata-project/exporter/blob/3.x/src/Bridge/Symfony/Resources/config/services.php#L64-L67
will be needed too.

See an example here:
https://github.com/sonata-project/SonataAdminBundle/blob/6dcebd2a72a3bea04adf48816330e6fa3031e5c7/src/Resources/config/core.php#L152-L157

So I think we'll need something like
```
$services->alias(ExporterInterface::class, 'sonata.exporter.exporter');
```